### PR TITLE
Log dagster_version and OS descriptor for telemetry data

### PR DESCRIPTION
**Pull request by dpeng817**
I might have overengineered a little bit here regarding dagster_version. Also not sure how kosher it is to store this information in a global state. Might be better to just get them functionally. I don't think that this should affect performance in any meaningful way, so maybe that's fine.

### CODE REVIEW

File: python_modules/dagster/dagster/core/telemetry.py [hunk 1]
```diff
@@ -42,6 +44,19 @@
...
+] = []  # cache dagster version, but due to circular imports we must wait to import it.
+OS_DESC = platform.platform()
+
+
+def _get_dagster_version():
+    if not DAGSTER_VERSION:
+        from dagster import __version__ as dagster_version
+
+        DAGSTER_VERSION.append(dagster_version)
+    return DAGSTER_VERSION[0]
```
Comment (alangenfeld) [inline python_modules/dagster/dagster/core/telemetry.py:58]: i dont think you need to cache this if its behind a function - shouldn't be expensive

**Review by alangenfeld — CHANGES_REQUESTED** [review]
to your queue

File: python_modules/dagster/dagster/core/telemetry.py [hunk 2]
```diff
@@ -42,6 +44,19 @@
 START_DAGIT_WEBSERVER = "start_dagit_webserver"
 DAEMON_ALIVE = "daemon_alive"
 TELEMETRY_VERSION = "0.2"
+DAGSTER_VERSION: List[
+    str
+] = []  # cache dagster version, but due to circular imports we must wait to import it.
+OS_DESC = platform.platform()
```
[the remark this answers is not in the document]
Comment (dpeng817) [reply python_modules/dagster/dagster/core/telemetry.py:50]: It's pretty verbose; I think terse in addition would be good

**Review by alangenfeld — APPROVED** [review]
noice

**Pull request merged**